### DISCUSSION TITLE
Casting?

### DISCUSSION THREAD

**Discussion by sunjaxx** [leinelissen/jellyfin-audio-player#65] [Q&A] [answered]
I read that casting (from android) is supported, but I can't find it in the app.... Am I overlooking it? :-) thanks!

**Comment by leinelissen** [comment, marked as answer]
Hi there! I am unsure of whether it works for Android natively, but at least for iOS it's not supported at this point. We are blocked by the library we are implementing here. If you want to check progress you can do so here: https://github.com/leinelissen/jellyfin-audio-player/issues/37. In the meanwhile, AirPlay is supported on iOS.
